.github/dependabot.yml

```diff
@@ -14,3 +14,27 @@ updates:
       - "scylladb/qa-maintainers"
     assignees:
       - "scylladb/qa-maintainers"
+
+  - package-ecosystem: "docker"
+    directories:
+      - "/defaults/docker_images/alternator-dns"
+      - "/defaults/docker_images/cassandra-stress"
+      - "/defaults/docker_images/cdc-stresser"
+      - "/defaults/docker_images/cql-stress-cassandra-stress"
+      - "/defaults/docker_images/gemini"
+      - "/defaults/docker_images/harry"
+      - "/defaults/docker_images/kcl"
+      - "/defaults/docker_images/latte"
+      - "/defaults/docker_images/nosqlbench"
+      - "/defaults/docker_images/ndbench"
```

Comment:
If it was meant to be in alphabetical order, then `ndbench` must be before `nosqlbench`.

Reply:
argue with copilot :)